tools/devops/automation/templates/common/load_configuration.yml

```diff
@@ -0,0 +1,106 @@
+# yamllint disable rule:line-length
+# This job will parse all the labels present in a PR, will set
+# the tags for the build AND will set a number of configuration
+# variables to be used by the rest of the projects
+parameters:
+
+  - name: uploadArtifacts
+    type: boolean
+    default: false
+
+  - name: use1ES
+    type: boolean
+    default: false
+
+  - name: repositoryAlias
+    type: string
+    default: self
+
+  - name: commit
+    type: string
+    default: HEAD
+
+  - name: uploadPrefix
+    type: string
+    default: '$(MaciosUploadPrefix)'
+
+  - name: testConfigurations
+    type: object
+    default: []
+
+  - name: supportedPlatforms
+    type: object
+    default: []
+
+  - name: testsLabels
+    type: string
+    default: ''
+
+  - name: statusContext
+    type: string
+    default: ''
+
+steps:
+  - template: checkout.yml
+    parameters:
+      isPR: true
+      repositoryAlias: ${{ parameters.repositoryAlias }}
+      commit: ${{ parameters.commit }}
+
+  - download: macios
+    displayName: Download Build Config
+    artifact: build-configuration
+
+  - pwsh: |
+      Get-ChildItem -Path "$(Pipeline.Workspace)/macios" -Recurse -Force
+    displayName: 'Display downloads'
+    timeoutInMinutes: 5
+
+  - bash: ./xamarin-macios/tools/devops/automation/scripts/bash/configure-platforms.sh
+    name: configure_platforms
+    displayName: 'Configure platforms'
+
+  - pwsh: |
+      Import-Module $Env:SYSTEM_DEFAULTWORKINGDIRECTORY/xamarin-macios/tools/devops/automation/scripts/MaciosCI.psd1
+      $jsonPath = Join-Path -Path "$(Build.ArtifactStagingDirectory)" -ChildPath "configuration.json"
+      Write-Host "##vso[task.setvariable variable=CONFIG_PATH]$jsonPath"
+      New-BuildConfiguration -ConfigFile $jsonPath
+    env:
+      GITHUB_TOKEN: $(GitHub.Token)
+      ACCESSTOKEN: $(AzDoBuildAccess.Token)
+    name: labels
+    displayName: 'Configure build'
+
+  - bash: ./xamarin-macios/tools/devops/automation/scripts/bash/configure-decisions.sh
+    name: decisions
+    displayName: 'Make decisions'
```

Comment:
This looks like a copy of configure.yml, and seems to create (and later upload) configuration.xml like configure.yml does. Isn't this file supposed to load the already created configuration.xml instead?

Reply:
It is a copy, the idea is that I'll be making changes one by one to keep the compatibility and remove those steps that are not needed. ATM we are loading the test matrix from the config, yet we are generating the config. That will be the next step.